app/views/parties/show.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1
+  = @party.respond_to?(:partyName) ? @party.partyName : ''
```

Comment:
this has been done. Please see new pull request for branch: aebirim/new-changes-party-controller